app/models/user.rb

```diff
@@ -0,0 +1,12 @@
+class User < ApplicationRecord
+  has_many :comments
+  has_many :likes
+  has_many :posts
```

Comment:
- [ ] @Yosefgeda great work adding the has_many relationship, however, you forgot to include the foreign_key referencing the user eg `has_many :posts, foreign_key: 'author_id'`, kindly consider adding foreign keys to make the association work.